misk-aws/src/main/kotlin/misk/dynamodb/DyEntity.kt

```diff
@@ -0,0 +1,14 @@
+package misk.dynamodb
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBTable
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBVersionAttribute
+
+/*
+ * This version field is used for optimistic locking.
+ */
+@DynamoDBTable(tableName = "")
```

Comment:
do abstract entities need table names?

Reply:
It doesn't need a actual name, but does need the table attribute for the member attributes to work